From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Mon, 17 Sep 2018 21:57:19 +0200
Subject: ungoogled-chromium: Always use local DevTools files instead of remote
 files from Google

NOTE: This can break Remote Debugging
This also fixes local debugging with domain substitution
Related comment: https://bugs.chromium.org/p/chromium/issues/detail?id=710701#c14
---
 third_party/blink/renderer/devtools/BUILD.gn              | 6 ++----
 .../blink/renderer/devtools/front_end/audits_worker.json  | 2 +-
 .../blink/renderer/devtools/front_end/devtools_app.json   | 8 ++++----
 third_party/blink/renderer/devtools/front_end/shell.json  | 2 +-
 .../blink/renderer/devtools/front_end/worker_app.json     | 2 +-
 5 files changed, 9 insertions(+), 11 deletions(-)

diff --git a/third_party/blink/renderer/devtools/BUILD.gn b/third_party/blink/renderer/devtools/BUILD.gn
--- a/third_party/blink/renderer/devtools/BUILD.gn
+++ b/third_party/blink/renderer/devtools/BUILD.gn
@@ -1621,11 +1621,9 @@ if (!external_devtools_frontend) {
           "$resources_out_dir/InspectorBackendCommands.js",
         ]
 
-    # Bundle remote modules in ChromeOS.
-    if (is_chromeos) {
-      grd_files += generated_remote_modules + devtools_emulated_devices_images +
+    # Always bundle remote modules.
+    grd_files += generated_remote_modules + devtools_emulated_devices_images +
                    lighthouse_locale_files
-    }
 
     inputs = grd_files + devtools_image_files
     outfile = "$root_gen_dir/devtools/devtools_resources.grd"
diff --git a/third_party/blink/renderer/devtools/front_end/audits_worker.json b/third_party/blink/renderer/devtools/front_end/audits_worker.json
--- a/third_party/blink/renderer/devtools/front_end/audits_worker.json
+++ b/third_party/blink/renderer/devtools/front_end/audits_worker.json
@@ -1,6 +1,6 @@
 {
     "modules": [
         { "name": "worker_service", "type": "autostart" },
-        { "name": "audits_worker", "type": "remote" }
+        { "name": "audits_worker" }
     ]
 }
diff --git a/third_party/blink/renderer/devtools/front_end/devtools_app.json b/third_party/blink/renderer/devtools/front_end/devtools_app.json
--- a/third_party/blink/renderer/devtools/front_end/devtools_app.json
+++ b/third_party/blink/renderer/devtools/front_end/devtools_app.json
@@ -4,23 +4,23 @@
     { "name": "inspector_main", "type": "autostart" },
     { "name": "mobile_throttling", "type": "autostart" },
 
-    { "name": "accessibility", "type": "remote" },
+    { "name": "accessibility" },
     { "name": "animation" },
     { "name": "audits" },
     { "name": "browser_debugger" },
     { "name": "css_overview" },
     { "name": "cookie_table" },
-    { "name": "dagre_layout", "type": "remote" },
+    { "name": "dagre_layout" },
     { "name": "devices" },
     { "name": "elements" },
-    { "name": "emulated_devices" , "type": "remote" },
+    { "name": "emulated_devices" },
     { "name": "har_importer" },
     { "name": "help" },
     { "name": "layers" },
     { "name": "layer_viewer" },
     { "name": "network" },
     { "name": "performance_monitor" },
-    { "name": "product_registry_impl", "type": "remote" },
+    { "name": "product_registry_impl" },
     { "name": "resources" },
     { "name": "security" },
     { "name": "timeline" },
diff --git a/third_party/blink/renderer/devtools/front_end/shell.json b/third_party/blink/renderer/devtools/front_end/shell.json
--- a/third_party/blink/renderer/devtools/front_end/shell.json
+++ b/third_party/blink/renderer/devtools/front_end/shell.json
@@ -21,7 +21,7 @@
 
     { "name": "changes" },
     { "name": "cm" },
-    { "name": "cm_modes", "type": "remote" },
+    { "name": "cm_modes" },
     { "name": "color_picker" },
     { "name": "console" },
     { "name": "coverage" },
diff --git a/third_party/blink/renderer/devtools/front_end/worker_app.json b/third_party/blink/renderer/devtools/front_end/worker_app.json
--- a/third_party/blink/renderer/devtools/front_end/worker_app.json
+++ b/third_party/blink/renderer/devtools/front_end/worker_app.json
@@ -10,7 +10,7 @@
     { "name": "help" },
     { "name": "layer_viewer" },
     { "name": "network" },
-    { "name": "product_registry_impl", "type": "remote" },
+    { "name": "product_registry_impl" },
     { "name": "resources" },
     { "name": "timeline" },
     { "name": "timeline_model" }
-- 
2.17.1

